UnitTests/BraintreePayPalTests/BTPayPalClient_Tests.swift

```diff
@@ -685,15 +685,19 @@ class BTPayPalClient_Tests: XCTestCase {
 
     func testTokenize_whenVaultRequest_setsVaultAnalyticsTag() async {
         let vaultRequest = BTPayPalVaultRequest()
+        payPalClient.webAuthenticationSession = MockWebAuthenticationSession()
```

Comment:
Can we do this in the setup instead?

Reply:
Also IMO I would rather the `postedIsVaultRequest` assertions just be tacked onto another test vs added a new test entirely for - but can go either way

Reply:
https://github.com/braintree/braintree_ios/pull/1324/commits/c522adfa0e78f3865008b9dc7ca3b04948bdf572

> Also IMO I would rather the postedIsVaultRequest assertions just be tacked onto another test vs added a new test entirely for - but can go either way

I don't have a strong preference, looks like new tests were created based on this comment: https://github.com/braintree/braintree_ios/pull/1296#discussion_r1606882074. If you'd rather, I can throw these assertions into existing tests.